runners/wikiConsole/templates/createDevon4ngProject.asciidoc

```diff
@@ -0,0 +1,15 @@
+== Create devon4ng project
+
+<% if (cdCommand) { %>
+
+=== Prerequisites
+
+As the project should be created at `<%= cdCommand; %>`, make sure that this directory exists before proceeding with the next step. 
+
```

Comment:
Maybe add ng for Prerequisites. Or can also check the installed_tools and if ng is not installed in the devonfw IDE and if missing it should be installed.